frontend/src/views/ShootList.vue

```diff
@@ -26,125 +26,13 @@ SPDX-License-Identifier: Apache-2.0
-                <v-list-item-action>
-                  <v-icon :color="checkboxColor(isFilterActive('progressing'))" v-text="checkboxIcon(isFilterActive('progressing'))"/>
-                </v-list-item-action>
-                <v-list-item-content class="grey--text text--darken-2">
-                  <v-list-item-title>Hide progressing clusters</v-list-item-title>
-                </v-list-item-content>
-              </v-list-item>
-              <v-list-item
-                @click.stop="toggleFilter('userIssues')"
-                :disabled="filtersDisabled"
-                :class="disabledFilterClass">
-                <v-list-item-action>
-                  <v-icon :color="checkboxColor(isFilterActive('userIssues'))" v-text="checkboxIcon(isFilterActive('userIssues'))"/>
-                </v-list-item-action>
-                <v-list-item-content class="grey--text text--darken-2">
-                  <v-list-item-title>Hide user issues</v-list-item-title>
-                </v-list-item-content>
-              </v-list-item>
-              <v-list-item
-                @click.stop="toggleFilter('deactivatedReconciliation')"
-                :disabled="filtersDisabled"
-                :class="disabledFilterClass">
-                <v-list-item-action>
-                  <v-icon :color="checkboxColor(isFilterActive('deactivatedReconciliation'))" v-text="checkboxIcon(isFilterActive('deactivatedReconciliation'))"/>
-                </v-list-item-action>
-                <v-list-item-content class="grey--text text--darken-2">
-                  <v-list-item-title>Hide clusters with deactivated reconciliation</v-list-item-title>
-                </v-list-item-content>
-              </v-list-item>
-              <v-list-item
-                @click.stop="toggleFilter('hideTicketsWithLabel')"
-                :disabled="filtersDisabled"
-                :class="disabledFilterClass"
-                v-if="!!gitHubRepoUrl && hideClustersWithLabels.length">
-                <v-list-item-action>
-                  <v-icon :color="checkboxColor(isFilterActive('hideTicketsWithLabel'))" v-text="checkboxIcon(isFilterActive('hideTicketsWithLabel'))"/>
-                </v-list-item-action>
-                <v-list-item-content class="grey--text text--darken-2">
-                  <v-list-item-title>
-                    Hide clusters with
-                    <v-tooltip top>
-                      <template v-slot:activator="{ on }">
-                        <span v-on="on"><tt>configured</tt><v-icon small>mdi-help-circle-outline</v-icon></span>
-                      </template>
-                      <div v-for="label in hideClustersWithLabels" :key="label">- {{label}}</div>
-                    </v-tooltip>
-                    ticket labels
-                  </v-list-item-title>
-                </v-list-item-content>
-              </v-list-item>
-            </template>
-          </v-list>
-        </v-menu>
+        <table-column-selection
```

Comment:
bug: I cannot select any columns under the table options menu

Reply:
not reducable